### PR TITLE
Make userdev version be fetched dynamically

### PR DESCRIPTION
**Pull request by Strokkur424**
Instead of hardcoding a version value or focing the user to fetch it itself (previous change to this site),  it should just be fetched live.

I have no idea what I am doing, so suggest changes freely

### CODE REVIEW

**Comment by Strokkur424** [comment]
No idea what the format issue is. If somebody can tell me, please do so

**Comment by zlataovce** [comment]
I'm not particularly fond of creating another component just for userdev, I'd much rather have it be a part of `VersionFormattedCode` (+ the placeholder should be documented in CONTRIBUTING.md)

**Comment by Strokkur424** [comment]
@zlataovce actually thinking about it, maybe I should put that util method into util/versionUtils.ts. Give me a second

**Comment by zlataovce** [comment]
> @zlataovce actually thinking about it, maybe I should put that util method into util/versionUtils.ts. Give me a second

yes, it should be cached in an expiring value

**Review by zlataovce — APPROVED** [review]

File: src/util/versionUtils.ts [hunk 1]
```diff
@@ -21,7 +21,6 @@ class ExpiringValue<T> {
     return this._value;
   }
 }
-
```
Comment (powercasgamer) [inline src/util/versionUtils.ts:24]: undo

**Review by powercasgamer — CHANGES_REQUESTED** [review]
otherwise looks *fine*

File: src/util/versionUtils.ts (hunk 1 above)
Comment (Strokkur424) [reply src/util/versionUtils.ts]: there

**Review by powercasgamer — APPROVED** [review]

**Review by olijeffers0n — APPROVED** [review]

**Pull request merged**